thunder/dynamo/utils.py

```diff
@@ -100,6 +100,14 @@ class ExampleInputMetaData:
     def stride(self) -> list[int]:
         return self.strides
 
+    def is_contiguous(self) -> bool:
```

Comment:
Instead of computing this, why not initialize this class with the contiguity information obtained from `torch.Tensor.is_contiguous`?